components/Workspaces/TrackedContributorsWizard/TrackedContributorsWizard.tsx

```diff
@@ -29,6 +30,8 @@ export const TrackedContributorsWizard = ({ onAddToTrackingList, onCancel }: Tra
   const [searchTerm, setSearchTerm] = useState<string | undefined>();
   const { data, isError, isLoading } = useSearchContributors(searchTerm);
 
+  const [pastedInput, setPastedInput, removePastedInput] = useLocalStorage("bulk-add-contributors", "");
```

Comment:
refactor (blocking): The `pastedInput` and `setPastedInput` variables from the destructured array are not used. Omit them.

```suggestion
const [, , removePastedInput] = useLocalStorage("bulk-add-repos", "");
```